### DISCUSSION TITLE
Not all go-e charger paremeters are available in HA

### DISCUSSION THREAD

**Discussion by ablansinger** [syssi/homeassistant-goecharger-mqtt#167] [Q&A] [answered]
I have installed a go-e changer and controller, and both are MQTT enabled and I see a lot of paramateres with MQTT.
I also see a lot of parameters in HA, but it seems they are just display values, and the "Device Variant" and "Firmware version" are not displayed as they are on the main page for this go-e integration. I do sea both a fwv="56.8" and a var=22 message with MQTT explorer.
The go-e integration indicates that it is version 0.25 and that it has one device and 110 entities.
Why do I not see the same parameters as in https://github.com/syssi/homeassistant-goecharger-mqtt?tab=readme-ov-file and why dont I see any settable parameters ?

**Comment by syssi** [comment]
Did you see the `Enabled per default` column at the table? The mentioned entities aren't enabled per default. You have to enable the entities manually:

1. Go to your Home Assistant "settings"
2. Select "Device & Services"
3. Switch to the "Devices" tab
4. Filter for "go-e" & open the Go-e charge device
5. The is a "Sensors" tile. Click "... entities not shown" to expand the list. 
6. Click on the gray "firmware version" entity
7. Click the gear icon on the entity model
8. Enable the entity using the `Enable` switch at the bottom.
9. Press `Update`

![enable-a-disabled-entity](https://github.com/user-attachments/assets/6991ed14-228e-4f19-9ece-4bc0df4cbde9)

**Reply by ablansinger** [reply, marked as answer]
I had noticeed that some was default not enabled, nut I had not found a way to enable them. Thank you.
